### ISSUE TITLE
[BUG] Profile in the Footer section is not working as expected

### ISSUE THREAD

**Issue by DurgaPrasad-R** [pooranjoyb/popShop#101]
The Profile navigation link in the footer section is navigating us to the home page rather than the profile page. This issue aims to fix this bug. I'm open to discuss if you have anything to clarify.
Screenshot:
![Screenshot (12)](https://github.com/pooranjoyb/popShop/assets/111352968/4eb98b19-bd1d-41ab-a9e6-fc3baee47b44)
Please assign me the issue under GSSoC 2024.

**Comment by pooranjoyb** [comment]
Congratulations, @DurgaPrasad-R! 🎉 Thank you for creating your issue. Your contribution is greatly appreciated and we look forward to working with you to resolve the issue. Keep up the great work!<p>We will promptly review your changes and offer feedback. Keep up the excellent work! Kindly remember to check our [contributing guidelines](https://github.com/pooranjoyb/popShop/blob/dev/CONTRIBUTING.md)</p>

**Comment by Mahitej28** [comment]
@DurgaPrasad-R this is not an issue rather the profile page is not been created hence for now its been redirected to the home page.

**Comment by DurgaPrasad-R** [comment]
@Mahitej28, I hope I'm not mistaken there does exist a profile page at '/home/profile'.
![image](https://github.com/pooranjoyb/popShop/assets/111352968/4948da9e-4b8b-49fc-94e8-43529660f198)

**Comment by Mahitej28** [comment]
> @Mahitej28, I hope I'm not mistaken there does exist a profile page at '/home/profile'. ![image](https://private-user-images.githubusercontent.com/111352968/331857680-4948da9e-4b8b-49fc-94e8-43529660f198.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTYxMTEzMjQsIm5iZiI6MTcxNjExMTAyNCwicGF0aCI6Ii8xMTEzNTI5NjgvMzMxODU3NjgwLTQ5NDhkYTllLTRiOGItNDlmYy05NGU4LTQzNTI5NjYwZjE5OC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNTE5JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDUxOVQwOTMwMjRaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT04NmQyODQ2NWQ1MDViYTE5NmUyMjVkYWEyZjk0YTFkNDFmMzU5NzRmOWUwZmY0YzNhZWU5MTg5YWUyOGJjNjE2JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.C7Q8ffW5AOrAWMpK82XJzBVR3ELdy0LBQ6Ggu-UuqAA)

yeah right but this page needs enhancement once it would be done we would route it profile page, this bug doesn't require a pr to be raised.

**Comment by DurgaPrasad-R** [comment]
@Mahitej28 I would like to work on the profile page.

**Comment by pooranjoyb** [comment]
Let's not take one line change for a PR @DurgaPrasad-R 
You can raise an issue for enhancement of the `Profile` page.
